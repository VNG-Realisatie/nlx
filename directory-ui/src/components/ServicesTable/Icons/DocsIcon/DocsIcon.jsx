// Copyright Â© VNG Realisatie 2018
// Licensed under the EUPL

import React from 'react'

const DocsIcon = () =>
  <svg width="24px" height="24px" viewBox="0 0 24 24">
    <g fillRule="nonzero">
      <path fill="currentColor" d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H17c1.1 0 2-.9 2-2V7l-5-5zM6 20V4h7l4 4v12H6z"/>
      <path fill="currentColor" d="M13 2h1l5 5v1h-6zM8 12h7v1H8v-1zm0 2h7v1H8v-1zm0 2h7v1H8v-1zm0-6h5v1H8v-1z"/>
    </g>
  </svg>

export default DocsIcon
