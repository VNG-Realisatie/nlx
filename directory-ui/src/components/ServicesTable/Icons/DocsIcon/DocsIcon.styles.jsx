import styled from 'styled-components'

const grey = '#CAD0E0'
const blue = '#517FFF'

export default styled.svg`
  display: inline-block
  width: 24px
  height: 24px
  fill: ${ p => p.color === 'blue' ? blue : grey };
`