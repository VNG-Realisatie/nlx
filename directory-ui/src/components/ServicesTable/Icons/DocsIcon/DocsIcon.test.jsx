import React from 'react'
import { shallow } from 'enzyme'
import DocsIcon from '../DocsIcon/DocsIcon';

it('should support the grey color', () => {
  expect(shallow(<DocsIcon color="grey"/>)).toMatchSnapshot()
})

it('should support the blue color', () => {
  expect(shallow(<DocsIcon color="blue"/>)).toMatchSnapshot()
})
