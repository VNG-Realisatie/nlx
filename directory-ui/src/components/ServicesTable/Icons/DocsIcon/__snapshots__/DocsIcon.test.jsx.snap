// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should support the blue color 1`] = `
<styled.svg
  color="blue"
  viewBox="0 0 24 24"
>
  <g
    fillRule="nonzero"
  >
    <path
      d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H17c1.1 0 2-.9 2-2V7l-5-5zM6 20V4h7l4 4v12H6z"
    />
    <path
      d="M13 2h1l5 5v1h-6zM8 12h7v1H8v-1zm0 2h7v1H8v-1zm0 2h7v1H8v-1zm0-6h5v1H8v-1z"
    />
  </g>
</styled.svg>
`;

exports[`should support the grey color 1`] = `
<styled.svg
  color="grey"
  viewBox="0 0 24 24"
>
  <g
    fillRule="nonzero"
  >
    <path
      d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H17c1.1 0 2-.9 2-2V7l-5-5zM6 20V4h7l4 4v12H6z"
    />
    <path
      d="M13 2h1l5 5v1h-6zM8 12h7v1H8v-1zm0 2h7v1H8v-1zm0 2h7v1H8v-1zm0-6h5v1H8v-1z"
    />
  </g>
</styled.svg>
`;

exports[`should support the offline status 1`] = `
<styled.svg
  viewBox="0 0 24 24"
>
  <g
    fillRule="nonzero"
  >
    <path
      d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H17c1.1 0 2-.9 2-2V7l-5-5zM6 20V4h7l4 4v12H6z"
    />
    <path
      d="M13 2h1l5 5v1h-6zM8 12h7v1H8v-1zm0 2h7v1H8v-1zm0 2h7v1H8v-1zm0-6h5v1H8v-1z"
    />
  </g>
</styled.svg>
`;

exports[`should support the online status 1`] = `
<styled.svg
  viewBox="0 0 24 24"
>
  <g
    fillRule="nonzero"
  >
    <path
      d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H17c1.1 0 2-.9 2-2V7l-5-5zM6 20V4h7l4 4v12H6z"
    />
    <path
      d="M13 2h1l5 5v1h-6zM8 12h7v1H8v-1zm0 2h7v1H8v-1zm0 2h7v1H8v-1zm0-6h5v1H8v-1z"
    />
  </g>
</styled.svg>
`;
